yarn.lock

```diff
@@ -1014,7 +1013,7 @@
   dependencies:
     regenerator-runtime "^0.13.2"
 
-"@babel/template@^7.1.0", "@babel/template@^7.1.2", "@babel/template@^7.2.2":
+"@babel/template@^7.1.0":
```

Comment:
What updated the lock file?

Reply:
I think I just ran yarn install after merging master, should I revert?